config/core/configmaps/autoscaler.yaml

```diff
@@ -137,3 +137,10 @@ data:
     # that should be used if none is specified. If omitted, the Knative
     # Horizontal Pod Autoscaler (KPA) is used by default.
     pod-autoscaler-class: "kpa.autoscaling.knative.dev"
+
+    # Scale to zero on deploy indicates whether new services will scale to
+    # 0 pods at the time of deployment. If minScale is set to > 0, new
+    # services will respect minScale and create the desired number of
+    # instances. If user provides readiness probe, readiness probe will
+    # still run and new services will not scale to 0.
```

Comment:
```suggestion
    # scale-to-zero-on-deploy creates new revisions with zero initial
    # replicas.  With this setting enabled the user container is not
    # activated, nor are readinessProbes evaluated before the new
    # Revision is declared Ready.  This setting should only be enabled
    # with extreme caution as it may cause traffic to shift to unverified
    # Revisions.  This behavior is overridden when the minScale
    # annotation is specified.
```

Reply:
New text is ok with me, but did you mean to drop the bit about minScale? Seemed like useful info.  If not in the end-user docs we definitely need to add it there too.

Reply:
I put in:
> This behavior is overridden when the minScale annotation is specified.

What do you think I missed?


Also to make sure I am reading things correctly, this was outdated, right?
> If user provides readiness probe, readiness probe will still run and new services will not scale to 0.

Reply:
oh geez, how did I miss that - sorry :-)

Yes, that sentence is old and should be removed if still there.

Reply:
Yep! This is now updated.